README.md

```diff
@@ -43,22 +110,39 @@ cd SmartSim-Scaling
 pip install -r requirements.txt
 ```
 
-You will need to install ``mpi4py`` in your python environment. The install instructions
-can be found by selecting [mpi4py docs](https://mpi4py.readthedocs.io/en/stable/install.html).
+> Note that if you are using a Cray machine, you will need to run `CC=cc CXX=CC pip install -r requirements.txt`.
 
-Lastly, the C++ applications themselves need to be built. One CMake edit is required.
-Near the top of the CMake file, change the path to the ``SMARTREDIS`` variable to
-the top level of the directory where you built or installed the SmartRedis library.
+Lastly, the C++ applications themselves need to be built. To complete this, 
+one CMake edit is required. When running `cmake ..`, 
+change the path to the ``SMARTREDIS`` variable to the top level of the directory 
+where you built or installed the SmartRedis library using the ``-DSMARTREDIS`` flag.
+An example of this is shown below.
 
-After the cmake edit, both tests can be built by running
+All tests can be built by running
 
 ```bash
-  cd cpp-<test name> # ex. cpp-inference for the inference tests
+  cd <language name>-<test name> # ex. cpp-inference for the cpp inference tests
   mkdir build && cd build
-  cmake ..
+  cmake .. -DSMARTREDIS=/path/to/SmartRedis
   make
 ```
 
+The app locations are shown below: 
```

Comment:
CMakeLists.txt isn't an app. Better to say "The CMake files used to build the various apps are shown below:"